src/Assets/Javascript/scroll-tracking.js

```diff
@@ -0,0 +1,40 @@
+function ScrollTracking($module) {
+  this.$module = $module;
+}
+
+const triggerAnalyticsEvent = (category, action) => {
+  ga("send", "event", {
+    eventCategory: category,
+    eventAction: action,
+    transport: "beacon",
+    nonInteraction: true
+  });
+};
+
+ScrollTracking.prototype.init = function() {
+  var $module = this.$module;
+
+  if (!$module) {
+    return;
+  }
+
+  document.addEventListener("scroll", () => {
+    const h = document.documentElement;
+    const b = document.body;
+    const st = "scrollTop";
+    const sh = "scrollHeight";
+    const pageUrl = window.location.href;
+
+    const percent = parseInt(((h[st] || b[st]) / ((h[sh] || b[sh]) - h.clientHeight)) * 100);
+    if (percent == 33) {
```

Comment:
Can the scroll event jump from `32` to `34`? Will it skip the event in this case?

Reply:
I think I have resolved this now :)